mkdocs.yml

```diff
@@ -133,6 +133,8 @@ nav:
     - Kserve: 
       - Load test KServe (HTTP): tutorials/integrations/kserve/kserve-http.md
       - Load test KServe (gRPC): tutorials/integrations/kserve/kserve-grpc.md
+      - Single Model validation: tutorials/integrations/kserve/kserve-custommetrics-one.md
```

Comment:
Could we please reorganize the kserve folder here so that we have the following structure?
```yaml
- Kserve:
  - Performance testing:
    - HTTP: <what this PR has now for HTTP load test>
    - gRPC: <what this PR has now for gRPC load test>
  - Canary testing: <what this PR has now for two models>
  - Continuous validation: <what this PR has now for single model>
  - Other (Iter8 v0.7): <what this PR has now for other>
```

Reply:
done